app/src/page/subscriptions/view.js

```diff
@@ -33,6 +35,11 @@ class SubscriptionsPage extends React.PureComponent {
     doFetchRecommendedSubscriptions();
   }
 
+  componentDidMount() {
+    const { doSetViewMode } = this.props;
+    doSetViewMode(Constants.SUBSCRIPTIONS_VIEW_ALL);
```

Comment:
Why not persist this value in redux? The app persists the entire subscriptions reducer. Then users won't have to move to latest only every time if that's what they want to see.

Reply:
Yeah, this makes sense. I'll add a setting option for this.